### ISSUE TITLE
Possibility to add more user metadata on page

### ISSUE THREAD

**Issue by jaydouble** [picklewagon/new-user-approve#54]
Hi,

at this moment the great plugin in only show the columns:
Username | Name | Mail address | Actions

I've checked the code, and those columns are hardcoded. 
It should be nice if there was a option to add more columns, or have a filter/hook to add more.

In this particular case I need to have the birthday of the user (which entered in the signup form) on this page, to match them to our members list to see if this user is already a member, or not. (I do have the birthday added in the normal user overview, but there are no links to accept/deny a new user.

Kind Regards.

**Comment by rommelvega25** [comment]
I agree. Need to have custom columns like for example Member ID and Address so that admin can verify more whether to Approve the user or not.
